versioning-policy.md

```diff
@@ -54,7 +54,7 @@ available APIs.
 
 In general, feature ("minor") releases occur about every 6 months. Hence, Spark 2.3.0 would
 generally be released about 6 months after 2.2.0. Maintenance releases happen as needed.
-A minor release usually sees 1-2 maintenance releases in the 6 months following its first
+A feature release usually sees a maintenance release in the 6 months following its first
```

Comment:
not sure "sees a" makes sense here, we usually do multiple

Maybe we can just get rid of this sentence or reword more like "Maintenance releases happen as needed in between the feature releases.